.ci/linux_devops_dotnet.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
Is installing dotnet explicitly needed?

Reply:
I've changed the yaml file so that it uses an explicit task to install the required dotnet version. We do need to specify the dotnet version we are using as there are multiple version available and we might not want to test against the default.